liquidator/liquidator.js

```diff
@@ -18,7 +18,8 @@ class Liquidator {
   queryAndLiquidate = async priceFeed => {
     Logger.info({
       at: "liquidator",
-      message: "Checking for under collateralized positions"
+      message: "Checking for under collateralized positions",
+      inputPrice: priceFeed
```

Comment:
How is this value used? I don't see it referenced in the slack push.

Reply:
it is used a few lines later in the call to the client with: 
```
const underCollateralizedPositions = this.empClient.getUnderCollateralizedPositions(priceFeed);
```
and is logged in slack. Again more formatting is required to improve the slack output but I will leave this for a seperate PR as mentioned here in issue #1031